Source/Behaviors/ITrigger.cs

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+namespace Dolittle.Assemblies
+{
+    /// <summary>
+    /// 
```

Comment:
Seems like there are some missing docs here.

Reply:
Ouch.. That snuck in there..   That was not supposed to be there.. Will clean up the code and at least have it in a better state.